src/bkl/plugins/gnu.py

```diff
@@ -210,13 +210,32 @@ def _linker_flags(self, toolset, target):
             cmd.append(LiteralExpr(toolset.pthread_ld_flags))
         return cmd
 
-    def commands(self, toolset, target, input, output):
-        cmd = [LiteralExpr("$(CXX) -o $@ $(LDFLAGS)"), input]
+
+    def _make_link_command(self, toolset, target, input, output_flags = None, extra_flags = None):
```

Comment:
The canonical style per PEP8 is without spaces around ` = ` here, i.e. `output_flags=None` etc: “Don't use spaces around the = sign when used to indicate a keyword argument or a default parameter value.”